src/attachments/types/relationship-filter.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum AttachmentRelationTargetType {
+  DATASET = "dataset",
```

Comment:
dataset -> is attachment of dataset
proposal -> is attachment of proposal
sample -> is attachment of sample

Reply:
Are you suggesting we should have AttachmentRelationType? The AttachmentRelationTargetType is used for targetType 
like:

```suggestion
export enum AttachmentRelationType{
  DATASET = "is attachment of dataset",
```

Reply:
I see. 
I think it would be more clear, although it is a nice to have at the moment